.gitattributes

```diff
@@ -9,4 +9,9 @@
 # https://stackoverflow.com/a/51564689 and
 # https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
 .yarn/** binary
-public/images/** binary
```

Comment:
Should we change this to `apps/prairielearn/public/images`?

Reply:
The only images in this directory are SVG files that will basically never change; I don't really see the value in treating them as binary?

Reply:
Oh right, I misread it and I was thinking this was vendoring those files.